python/semantic_kernel/ai/open_ai/services/azure_text_embedding.py

```diff
@@ -32,13 +57,14 @@ def __init__(
 
         self._endpoint = endpoint
         self._api_version = api_version
+        self._api_type = "azure_ad" if use_ad_auth else "azure"
```

Comment:
use_ad_auth is a boolean here. Should this be "ad_auth"?

Reply:
Renamed to `ad_auth`